src/ast2ram/provenance/SubproofGenerator.cpp

```diff
@@ -129,7 +129,8 @@ Own<ram::Operation> SubproofGenerator::addBodyLiteralConstraints(
             auto constraint = mk<ram::Constraint>(
                     BinaryConstraintOp::EQ, std::move(lhs), mk<ram::SubroutineArgument>(i));
             op = mk<ram::Filter>(std::move(constraint), std::move(op));
-        } else if (const auto* adt = as<ast::BranchInit>(arg)) {
+        } else if ([[maybe_unused]] const auto* adt = as<ast::BranchInit>(arg)) {
+            (void)adt;  // GCC 9.2.1 is stupid and ugly and isn't honouring `[[maybe_unused]]`
```

Comment:
Perhaps we rephrase comment to "gcc 9.2.1 does not support [[maybe_unused]]"

Reply:
I'll interpret this as 'be tactful'.
Pedantic: The odd thing is that it *does* support `[[maybe_unused]]`, it just doesn't appear to work correctly in the specific case of the vardecl being in an `if` conditional, hence the ire.